src/io/flutter/run/FlutterAppState.java

```diff
@@ -103,6 +105,13 @@ protected ProcessHandler startProcess() throws ExecutionException {
       }
     }
 
+    if (device != null) {
+      // Bring simulator to front.
+      if (device.emulator() && Objects.equals(device.platform(), "ios")) {
+        new OpenSimulatorAction(true).actionPerformed(null);
```

Comment:
Did a bit of poking around and I think this is the recommended approach.  Would like to avoid `osascript` if possible but maybe that's just superstition.  Let's see if this has any hiccups and update if needed.

Thanks!